org.eclipse.scout.rt.app/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2010, 2023 BSI Business Systems Integration AG
+  ~
+  ~ This program and the accompanying materials are made
+  ~ available under the terms of the Eclipse Public License 2.0
+  ~ which is available at https://www.eclipse.org/legal/epl-2.0/
+  ~
+  ~ SPDX-License-Identifier: EPL-2.0
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.eclipse.scout.rt</groupId>
+    <artifactId>org.eclipse.scout.rt</artifactId>
+    <version>24.1-SNAPSHOT</version>
+    <relativePath>../org.eclipse.scout.rt</relativePath>
+  </parent>
+
+  <artifactId>org.eclipse.scout.rt.app</artifactId>
+  <packaging>jar</packaging>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.eclipse.scout.rt</groupId>
+      <artifactId>org.eclipse.scout.rt.jetty</artifactId>
+    </dependency>
+    <dependency>
+      <!-- due to org.eclipse.scout.rt.server.commons.WebappEventListener -->
```

Comment:
Not (yet) at least, because Tomcat deployments will not (and shouldn't) have a dependency to the Jetty module, still requiring the WebappEventListener though. I'll leave a comment in the Application class that it might be moved if Tomcat support is dropped completely.